src/pymor/models/interface.py

```diff
@@ -352,15 +352,14 @@ def compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu
             self.logger.info(f'Solving {self.name} for {mu} ...')
 
         # first call _compute to give subclasses more control
-        data = self._compute(solution=solution, output=output,
+        data = self._compute(solution=solution, output=output, input=input,
```

Comment:
Maybe this is not necessary because `mu` contains `input`?

Reply:
Oh yes.